Expecto/Logging.fs

```diff
@@ -879,7 +879,7 @@ module internal ANSIOutputWriter =
     override __.WriteLine() = write "\n"
 
   let mutable internal colours = None
-  let internal setColourLevel c = if colours.IsNone then colours <- Some c
+  let internal setColourLevel c = colours <- Some c
```

Comment:
I couldn't discern a reason this needed to be settable only once and it prevents the colour setting from working as expected in interactive environments

Reply:
#481